app/javascript/sdk/IFrameHelper.js

```diff
@@ -43,6 +43,15 @@ const updateAuthCookie = cookieContent =>
     sameSite: 'Lax',
   });
 
+const updateCampaignReadStatus = () => {
+  const expireBy = new Date();
+  expireBy.setHours(expireBy.getHours() + 1);
```

Comment:
This method is not reliable in day light savings timezone. Please see the thread [here](https://stackoverflow.com/questions/1050720/how-to-add-hours-to-a-date-object#comment102500929_1051641).

Use date-fns and add hours instead.

Reply:
Thanks for the suggestion. I have added it 👍